openquake/qa_tests_data/classical/case_15/expected/hazard_uhs-mean.csv

```diff
@@ -1,4 +1,4 @@
 lon,lat,poe~0.01:2,poe~0.1:2,poe~0.2:2
--1.00000001E-01,0.00000000E+00,2.00000003E-01 8.00000012E-01,2.00000003E-01 8.00000012E-01,2.00000003E-01 6.34829700E-01
-0.00000000E+00,0.00000000E+00,2.00000003E-01 8.00000012E-01,2.00000003E-01 8.00000012E-01,2.00000003E-01 5.94248533E-01
-1.00000001E-01,0.00000000E+00,2.00000003E-01 8.00000012E-01,2.00000003E-01 8.00000012E-01,2.00000003E-01 7.47483134E-01
+-1.00000001E-01,0.00000000E+00,8.00000012E-01 8.00000012E-01,8.00000012E-01 8.00000012E-01,6.34829700E-01 6.34829700E-01
```

Comment:
Similar comment as above; expected values for poe~0.01 seem to have changed here.